### DISCUSSION TITLE
[30 Giugno 2022] Modifica CPU

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/architettura-degli-elaboratori#72] [Modifiche architettura MIPS (M-Z)]
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/b93529e3-c627-4225-b60d-450a14f4a9b7)
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/b17c0cfd-b3b5-4040-9401-c321c63d4825)
![CPU](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/3097b788-918c-42c9-9f92-4fc22caa7837)

**Comment by daromacarlo** [comment]
![1](https://github.com/sapienzastudentsnetwork/archelab2223/assets/135441131/684e7eb6-a4a3-4893-ab61-44679f409334)


![12](https://github.com/sapienzastudentsnetwork/archelab2223/assets/135441131/09e93b05-1ad5-4bbe-b75c-a579af631569)

**Comment by daromacarlo** [comment]
il periodo di clock dovrebbe essere 300Ns non 275.

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/sapienzastudentsnetwork/archelab2223/assets/79638284/290c1159-d4f3-4ae9-b39d-cd2cea6c148e)
